contrib/akamai/controlbar/ControlBar.js

```diff
@@ -619,12 +625,12 @@ var ControlBar = function (dashjsMediaPlayer, displayUTCTimeCodes) {
         return (sameId && sameViewpoint && sameLang && sameRoles && sameAccessibility && sameAudioChannelConfiguration);
     };
 
-    var getMenuContent = function (type, arr, contentFunc) {
+    var getMenuContent = function (type, arr, contentFunc, autoswitch = true) {
```

Comment:
I like the idea of setting a default value for autoswitch. However, I fear that it might break on older platforms/browsers like IE11 or embedded devices since we do not transpile Controlbar.js from ES6 to ES5. Can we move this to the function itself.